src/test/javascript/spec/component/shared/sidebar/sidebar-card-item.component.spec.ts

```
@@ -37,4 +37,60 @@ describe('SidebarCardItemComponent', () => {
         const compiled = fixture.debugElement.nativeElement;
         expect(compiled.querySelector('#test-sidebar-card-title').textContent).toContain(testItem.title);
     });
+
+    it('should remove the correct prefix from the name when groupKey is in channelTypes', () => {
+        const prefixes = ['exercise-', 'lecture-', 'exam-'];
+        const channelTypes = ['exerciseChannels', 'lectureChannels', 'examChannels'];
+
+        for (let i = 0; i < prefixes.length; i++) {
+            const prefix = prefixes[i];
+            const groupKey = channelTypes[i];
+            const nameWithPrefix = prefix + 'TestName';
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)